x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/rule_preview/preview_table_cell_renderer.tsx

```diff
@@ -5,11 +5,22 @@
  * 2.0.
  */
 
-import type React from 'react';
+import React from 'react';
 import type { EuiDataGridCellValueElementProps } from '@elastic/eui';
+import { TableId } from '@kbn/securitysolution-data-table';
 import type { CellValueElementProps } from '../../../../../common/types';
+import { SourcererScopeName } from '../../../../common/store/sourcerer/model';
 import { RenderCellValue } from '../../../../detections/configurations/security_solution_detections';
 
 export const PreviewRenderCellValue: React.FC<
   EuiDataGridCellValueElementProps & CellValueElementProps
-> = (props) => RenderCellValue({ ...props, asPlainText: true });
+> = (props) => {
+  return (
+    <RenderCellValue
+      {...props}
+      asPlainText={true}
+      scopeId={SourcererScopeName.detections}
+      tableId={TableId.rulePreview}
+    />
+  );
+};
```

Comment:
Nit: unnecessary `return`

```suggestion
> = (props) => (
  <RenderCellValue
    {...props}
    asPlainText={true}
    scopeId={SourcererScopeName.detections}
    tableId={TableId.rulePreview}
  />
);
```

Reply:
I must be the only one within kibana who prefers this way ha. It's more explicit, and also future me appreciates being able to console.log(props) on the line above the return without having to insert brackets and parentheses.